conf/messages.en

```diff
@@ -484,30 +484,28 @@ updateOtherIndividualsClosing.additionalContent2 = Right now HMRC will only ask
 addNonEeaCompany.title = Do you need to add a controlling interest in a Non-EEA company?
 addNonEeaCompany.heading = Do you need to add a controlling interest in a Non-EEA company?
 addNonEeaCompany.paragraph1 = A non-EEA company is a company registered outside the following countries:
-addNonEeaCompany.bullet1 = The EU
+addNonEeaCompany.bullet1 = all countries within the European Union (EU)
 addNonEeaCompany.bullet2 = Iceland
 addNonEeaCompany.bullet3 = Lichtenstein
 addNonEeaCompany.bullet4 = Norway
-addNonEeaCompany.bullet5 = The United Kingdom
 addNonEeaCompany.paragraph2 = The trust will have a controlling interest in a non-EEA company if any of the following apply:
-addNonEeaCompany.bullet6 = the trustees hold more than 50 percent of the shares in the company
-addNonEeaCompany.bullet7 = the trustees hold more than 50 percent of the voting rights in the company
-addNonEeaCompany.bullet8 = the trustees hold the right to appoint or remove a majority of the board of directors of the company
-addNonEeaCompany.bullet9 = the trustees can exercise significant influence or control over the company
+addNonEeaCompany.bullet5 = the trustees hold more than 50 percent of the shares in the company
```

Comment:
The still seems to still have the first list showing options 1 to 5, and the second showing 6 to 9. 
Should this not also be changed